pages/common/aws-codecommit.md

```diff
@@ -0,0 +1,24 @@
+# aws codecommit
+
+> AWS CodeCommit is a secure, highly scalable, managed source control service that hosts private Git repositories.
+> More information: <https://aws.amazon.com/codecommit/>.
```

Comment:
```suggestion
> More information: <https://docs.aws.amazon.com/cli/latest/reference/codecommit>.
```
We prefer linking to documentation directly.

Reply:
@Abhay5855 is this pending?